specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-02-09/examples/Quotas/Get.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parameters": {
+        "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+        "location": "local",
+        "quotaName": "testQuotaCreateUpdateDelete",
+        "api-version": "2018-02-09"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/8158498d-27b1-4ccf-9aa1-de0f925731e6/providers/Microsoft.Compute.Admin/locations/local/quotas/testQuotaCreateUpdateDelete",
+                "name": "testQuotaCreateUpdateDelete",
```

Comment:
Name and type should be consistent. In this case both are wrong. Name should be local/testQ..., and type should start with the namespace.